cloudflared/rootfs/etc/cont-init.d/10-cloudflared-config.sh

```diff
@@ -400,10 +514,17 @@ main() {
             bashio::exit.ok
         fi
     fi
+
+    if bashio::config.true 'warp_reset' ; then
+        warp_reset
+    fi
+
     createConfig
 
     createDNS
 
+    createRoute
```

Comment:
I think we should stay in the same pattern here and check the config before calling the method and not check in the method "createRoute" itself and return if not set. Obviously works both, it should just be the same for all the options.